### PR TITLE
Mig mgmtsystem audit to v. 10.0

### PR DESCRIPTION
**Pull request by eugen-don**
Mig mgmtsystem audit to v. 10.0

Depends on:

- [x] #169 mgmtsystem nonconformity

### CODE REVIEW

File: mgmtsystem_audit/models/mgmtsystem_audit.py [hunk 1]
```diff
@@ -1,28 +1,11 @@
...
-
-
-class MgmtSystemAudit(models.Model):
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api, _
+
+
+class MgmtsystemAudit(models.Model):
```
Comment (angelmoya) [inline mgmtsystem_audit/models/mgmtsystem_audit.py:9]: Why you change MgmtsystemAudit CamelCase?

**Review by angelmoya — COMMENTED** [review]
I will test after #169

File: mgmtsystem_audit/models/mgmtsystem_audit.py (hunk 1 above)
Comment (eugen-don) [reply mgmtsystem_audit/models/mgmtsystem_audit.py:9]: Each letter that follows a fullstop, has to be written in CamesCase so:

model mgmtsystem.audit = MgmtsystemAudit
and model mgmtsystem.nonconformity = MgmtsystemNonconformity
and model mgmtsystem.nonconformity.stage = MgmtsystemNonconformityStage

thats how id like it to be in the future to stay consistent with the code in the other modules

**Comment by eugen-don** [comment]
HI @angelmoya 
work is done... could you test?

**Review by max3903 — APPROVED** [review]
:+1:

**Review by dreispt — APPROVED** [review]
LGTM

**Comment by angelmoya** [comment]
Nice to see this quick review :+1:  I will check next PR

**Pull request merged**